src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -40,7 +41,7 @@ public class CustomFinder : IFindSagas<TestSaga08.SagaData08>.Using<SomeOtherMes
                 // ReSharper disable once MemberCanBePrivate.Global
                 public Context Context { get; set; }
 
-                public Task<TestSaga08.SagaData08> FindBy(SomeOtherMessage message, ReadOnlyContextBag context)
+                public Task<TestSaga08.SagaData08> FindBy(SomeOtherMessage message, SynchronizedStorageSession storageSession, ReadOnlyContextBag context)
```

Comment:
Here it would make sense but not for handlers. Should we have an explicit `SagaFinderContext`? I called YAGNI on that since it is only two arguments (apart from message) and we called YAGNI on some other similar 2-value wrapping contexts (AFAIR).